app/routes/about/components/leaders-grid.component.tsx

```diff
@@ -1,46 +1,96 @@
+import { Icon } from "~/primitives/icon/icon";
 import { leaders } from "./leaders-data";
 import { cn } from "~/lib/utils";
+import Modal from "~/primitives/Modal";
+import { useEffect, useState } from "react";
+import { LeadersModal } from "~/components/modals/leaders";
+import { Author } from "~/routes/author/loader";
+import { useFetcher } from "react-router";
 
 export function LeaderGrid() {
   return (
     <div className="flex items-start lg:items-end gap-3">
       {leaders.map((leader, index) => (
-        <div
-          key={leader.name}
-          className={cn(
-            "group w-full",
-            "md:w-[50%]",
-            "lg:w-[25%]",
-            index === 0 && "lg:w-[28%]",
-            "cursor-pointer"
-          )}
-        >
-          <div className="relative mb-6">
-            <div className="overflow-hidden rounded-[8px]">
-              <img
-                src={leader.imagePath}
-                alt={leader.name}
-                className={cn(
-                  "w-full aspect-[32/46] object-cover",
-                  "transform transition-transform duration-300 group-hover:scale-105"
-                )}
+        <LeaderCard key={index} index={index} leader={leader} />
+      ))}
+    </div>
+  );
+}
+
+const LeaderCard = ({ leader, index }: { leader: any; index: number }) => {
+  const [openModal, setOpenModal] = useState(false);
+  const [author, setAuthor] = useState<Author | null>(null);
+  const fetcher = useFetcher();
+  const formData = new FormData();
+  formData.append("id", leader.id);
+
+  useEffect(() => {
+    try {
+      fetcher.submit(formData, {
+        method: "post",
```

Comment:
I'm a bit confused here. Shouldn't this be a `fetcher.load()` function utilizing the `/author` route? Since we are just loading in the author data?